libafl/src/mutators/scheduled.rs

```diff
@@ -203,6 +203,38 @@ where
     }
 }
 
+/// Tuple type of the mutations that compose the Havoc mutator without crossover mutations
+pub type HavocMutationsNoCrossoverType = tuple_list_type!(
```

Comment:
we could move these sub-lists to the main library if we want

Reply:
How do you mean? They're in the main library now, right?

Reply:
Ah nevermind I'm stupid